package.json

```diff
@@ -23,7 +23,8 @@
     "pg": "^4.4.3",
     "ramda": "^0.19.0",
     "request": "^2.67.0",
-    "slack-client": "^1.5.1"
+    "slack-client": "^1.5.1",
+    "wordwrap": "^1.0.0"
```

Comment:
Pretty sure it's some mismatch with the shrinkwrap file, but I am honestly not sure.